packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -226,10 +273,37 @@ export class TriggerManager extends Component {
     if (typeof onError === 'function') onError(error)
   }
 
-  handleCipherSelect() {
-    this.setState({
-      step: 'accountForm'
-    })
+  handleCipherSelect(selectedCipher) {
+    const { konnector } = this.props
+    const { fields } = konnector
+    const sanitizedFields = manifest.sanitizeFields(fields)
+    const account = this.cipherToAccount(selectedCipher)
+    const values = manifest.getFieldsValues(konnector, account)
+
+    const requiredFields = Object.entries(sanitizedFields)
+      .filter(([, value]) => value.required)
+      .map(([key]) => key)
+
+    const hasValuesForRequiredFields =
+      intersection(Object.keys(values), requiredFields).length ===
+      requiredFields.length
```

Comment:
Should we move those previous lines to an helper? It seems that they are not a React related stuff, but more "konnector/account"?

Reply:
Done in https://github.com/cozy/cozy-libs/pull/782/commits/5a29d9fb712937e6095b2ce0b2d92a86f6aee79f